src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
I also just realised that you talked about general headings for all parts in ILIAS, while I talked about how grouping in the listings could be done. Do we have this "only expand one group" behaviour there too? Where can I observe it, because I didn't, so far. Do we have this "multi level nesting" there too? I also did not observe it there till now.
Until now, we also always somehow internalized the headings into the component (see Cards, Decks, Panels) by turning it into a "title"-property. I now see that we could of course model this differently by using a separate "heading" component. I think we didn't do this, since headings didn't have properties on their own for the things we modeled so far.